### PR TITLE
[doc][trans] submit translation for Yi quick start - choose your path and Yi quick start - pip

### LINKED ISSUE #320: [Translation Task] Yi README: Quick start (choose your path + pip)
> 该 issue 为「Yi 翻译计划」第 1 期 任务 issue，欢迎你参与共建~ 🙌

# 任务介绍

- 翻译方向：英翻中

- 待译原文：Yi README:  
  - [Quick start  - Choose your path](https://github.com/01-ai/Yi?tab=readme-ov-file#choose-your-path)
  - [Quick start  - pip](https://github.com/01-ai/Yi?tab=readme-ov-file#pip)

- 翻译步骤：参阅[「Yi 翻译计划」第 1 期](https://github.com/01-ai/Yi/discussions/314)。

# 任务要求

- 了解 [git 基本操作](https://www.cnblogs.com/daniel-hutao/p/open-a-pr-in-github.html)。

- 译文需要技术准确无误、表达流畅。

# 参考示例
- Translation and review: https://github.com/01-ai/Yi/pull/331

### PR DESCRIPTION
**Pull request by 202030481266**
This PR resolves #320

### CODE REVIEW

File: README_CN.md [hunk 1]
```diff
@@ -264,94 +264,94 @@ sequence length and can be extended to 32K during inference time.
...
+ ![快速开始 - 选择你的路线](https://github.com/01-ai/Yi/blob/main/assets/img/quick_start_path.png)
 
-#### 🎯 Deploy Yi locally
+#### 🎯 在本地部署Yi
 
-If you prefer to deploy Yi models locally, 
+如果你更喜欢在本地部署Yi模型，
 
-  - 🙋‍♀️ and you have **sufficient** resources (for example, NVIDIA A800 80GB), you can choose one of the following methods:
+  - 🙋‍♀️ 并且你有足够的资源（例如，NVIDIA A800 80GB），你可以选择以下方法之一：
```
Comment (markli404) [inline README_CN.md:281]: 足够需要加粗

Comment (markli404) [inline README_CN.md:281]: 足够需要加粗

File: README_CN.md [hunk 2]
```diff
@@ -264,94 +264,94 @@ sequence length and can be extended to 32K during inference time.
...
-##### 🙋‍♀️ Run Yi with APIs
+##### 🙋‍♀️ 通过API来使用Yi
 
-If you want to explore more features of Yi, you can adopt one of these methods:
+如果你想探索Yi的更多功能，你可以采用以下方法之一：
 
-- Yi APIs (Yi official)
-  - [Early access has been granted](https://x.com/01AI_Yi/status/1735728934560600536?s=20) to some applicants. Stay tuned for the next round of access!
+- Yi APIs (Yi官方)
+  - [早期申请](https://x.com/01AI_Yi/status/1735728934560600536?s=20)的部分申请者已经获得了访问权限。敬请期待下一轮访问申请的信息！
```
Comment (markli404) [inline README_CN.md:297]: 早期访问，或者抢先体验

File: README_CN.md [hunk 3]
```diff
@@ -264,94 +264,94 @@ sequence length and can be extended to 32K during inference time.
...
+本教程将指导你在**A800（80G）上本地运行Yi-34B-Chat**，并执行推理。
 
-#### Step 0: Prerequistes
+#### Step 0: 前提条件
  
-- Make sure Python 3.10 or a later version is installed.
+- 确保安装了Python 3.10或更高版本。
 
-- If you want to run other Yi models, see [software and hardware requirements](#deployment)
+- 如果你想运行其他Yi模型，请参阅[部署要求](#deployment)
```
Comment (markli404) [inline README_CN.md:328]: 这个超链接需要更改

File: README_CN.md [hunk 4]
```diff
@@ -264,94 +264,94 @@ sequence length and can be extended to 32K during inference time.
 - [Deployment](https://github.com/01-ai/Yi/blob/main/docs/deployment.md)
 - [Learning hub](https://github.com/01-ai/Yi/blob/main/docs/learning_hub.md)
 
-## Quick start
+## 快速开始
```
Comment (markli404) [inline README_CN.md:267]: 上面的目录也需要翻译

File: README_CN.md (hunk 3 above)
Comment (202030481266) [reply README_CN.md:328]: 这个超链接的话，我将它改为 **部署**， 因为它还没有被翻译

**Comment by 202030481266** [comment]
fix translation according to review suggestions

File: README_CN.md [hunk 5]
```diff
@@ -254,104 +254,104 @@ sequence length and can be extended to 32K during inference time.
...
+如果你更喜欢在本地部署Yi模型，
 
-  - 🙋‍♀️ and you have **sufficient** resources (for example, NVIDIA A800 80GB), you can choose one of the following methods:
+  - 🙋‍♀️ 并且你有**足够**的资源（例如，NVIDIA A800 80GB），你可以选择以下方法之一：
     - [pip](#pip)
     - [Docker](https://github.com/01-ai/Yi/blob/main/docs/README_legacy.md#11-docker)
     - [conda-lock](https://github.com/01-ai/Yi/blob/main/docs/README_legacy.md#12-local-development-environment)
 
-  - 🙋‍♀️ and you have **limited** resources (for example, a MacBook Pro), you can use [llama.cpp](#quick-start---llamacpp)
+  - 🙋‍♀️ 但是你的资源很有限（例如，一台MacBook Pro），你可以使用[llama.cpp](#quick-start---llamacpp)
```
Comment (lljzhgxd) [inline README_CN.md:286]: quick-start也需要翻译

File: README_CN.md [hunk 6]
```diff
@@ -254,104 +254,104 @@ sequence length and can be extended to 32K during inference time.
...
 
-If you want to chat with Yi with more customizable options (e.g., system prompt, temperature, repetition penalty, etc.), you can try one of the following options:
+如果你想要与Yi进行聊天，并且有更多可定制的选项（例如，系统提示、温度、重复惩罚等），你可以尝试以下选项之一：
   
-  - [Yi-34B-Chat-Playground](https://platform.lingyiwanwu.com/prompt/playground) (Yi official)
-    - Access is available through a whitelist. Welcome to apply (fill out a form in [English](https://cn.mikecrm.com/l91ODJf) or [Chinese](https://cn.mikecrm.com/gnEZjiQ)).
+  - [Yi-34B-Chat-Playground](https://platform.lingyiwanwu.com/prompt/playground) (Yi官方)
+    - 访问是通过白名单实现的。欢迎申请（填写[英文申请表](https://cn.mikecrm.com/l91ODJf) 或者 [中文申请表](https://cn.mikecrm.com/gnEZjiQ)）。
   
   - [Yi-34B-Chat-Playground](https://replicate.com/01-ai/yi-34b-chat) (Replicate)
```
Comment (lljzhgxd) [inline README_CN.md:308]: Replicate也需要翻译

File: README_CN.md [hunk 7]
```diff
@@ -254,104 +254,104 @@ sequence length and can be extended to 32K during inference time.
...
 
-If you want to explore more features of Yi, you can adopt one of these methods:
+如果你想探索Yi的更多功能，你可以采用以下方法之一：
 
-- Yi APIs (Yi official)
-  - [Early access has been granted](https://x.com/01AI_Yi/status/1735728934560600536?s=20) to some applicants. Stay tuned for the next round of access!
+- Yi APIs (Yi官方)
+  - [早期访问](https://x.com/01AI_Yi/status/1735728934560600536?s=20)的部分申请者已经获得了访问权限。敬请期待下一轮访问申请的信息！ 
 
 - [Yi APIs](https://replicate.com/01-ai/yi-34b-chat/api?tab=nodejs) (Replicate)
```
Comment (lljzhgxd) [inline README_CN.md:299]: Replicate也需要翻译

File: README_CN.md [hunk 8]
```diff
@@ -71,9 +71,9 @@
     - [Other info](#other-info)
   - [🎉 News](#-news)
 - [🟢 How to use Yi?](#-how-to-use-yi)
```
Comment (lljzhgxd) [inline README_CN.md:73]: 这些目录也需要翻译

**Review by lljzhgxd — COMMENTED** [review]
已提交审校

**Comment by 202030481266** [comment]
fix translation according to suggestion by @lljzhgxd

**Comment by GloriaLee01** [comment]
@202030481266  We would like to invite you to please include **the prompt used for translation in your PR description**, specifically mentioning that you are using the [Yi-34B-Chat](https://huggingface.co/spaces/01-ai/Yi-34B-Chat) model. This would greatly assist us in better understanding the context and providing the necessary support.
We appreciate your cooperation and participation!   🥳

**Comment by 202030481266** [comment]
Ok, I use Yi-chat-34B model to assist me with the translation tasks. Here is my prompt:

image
image
continue talking with :
image

**Comment by 202030481266** [comment]
Ok, I use Yi-chat-34B model to assist me with the translation tasks. Here is my prompt:

image

image

continue talking with :

image

Basically the prompt is:
你是一名01ai的高级文档工程师，你非常擅长Markdown语言，并且擅长英文和中文。下面是一段Markdown笔记，你需要将里面的英文翻译为中文同时保持笔记的格式正确（注意链接不需要翻译！）。

>>>
the content you need to translate
>>>

请翻译：

**Comment by GloriaLee01** [comment]
@202030481266  It seems that your translation is completed. Congratulations! 🥳 
If the translation is completed, you can **resolve all the conversations** in both the comments section and the PR (Pull Request) section.

File: README_CN.md [hunk 9]
```diff
@@ -252,106 +252,106 @@ sequence length and can be extended to 32K during inference time.
...
-##### 🙋‍♀️ Run Yi in playground
+##### 🙋‍♀️ 在游乐场使用 Yi
 
-If you want to chat with Yi with more customizable options (e.g., system prompt, temperature, repetition penalty, etc.), you can try one of the following options:
+如果你想要与 Yi 进行聊天，并且有更多可定制的选项（例如，系统提示、温度、重复惩罚等），你可以尝试以下选项之一：
   
-  - [Yi-34B-Chat-Playground](https://platform.lingyiwanwu.com/prompt/playground) (Yi official)
-    - Access is available through a whitelist. Welcome to apply (fill out a form in [English](https://cn.mikecrm.com/l91ODJf) or [Chinese](https://cn.mikecrm.com/gnEZjiQ)).
+  - [Yi-34B-Chat-Playground](https://platform.lingyiwanwu.com/prompt/playground) (Yi 官方)
+    - 访问是通过白名单实现的。欢迎申请（填写[英文申请表](https://cn.mikecrm.com/l91ODJf) 或者 [中文申请表](https://cn.mikecrm.com/gnEZjiQ)）。
```
Comment (GloriaLee01) [inline README_CN.md:306]: 访问是通过白名单实现的。→  如果你申请加入白名单，就可以使用交互式平台

**Comment by GloriaLee01** [comment]
@202030481266 I shared some views in the PR section. Please feel free to give feedback. Thanks! 🥳
Once the translation is completed, you can close all the discussions in both the comments section and the PR section.

**Comment by 202030481266** [comment]
I have already resolve all the conversations. @GloriaLee01

**Review by GloriaLee01 — APPROVED** [review]

File: README_CN.md [hunk 10]
```diff
@@ -252,106 +252,106 @@ sequence length and can be extended to 32K during inference time.
...
   
-  - [Yi-34B-Chat-Playground](https://replicate.com/01-ai/yi-34b-chat) (Replicate) 
+  - [Yi-34B-Chat-Playground](https://replicate.com/01-ai/yi-34b-chat) (Replicate，第三方网站) 
 
-##### 🙋‍♀️ Chat with Yi
+##### 🙋‍♀️ 与 Yi 聊天
 
- If you want to chat with Yi, you can use one of these online services, which offer a similar user experience:
+如果你想要与 Yi 聊天，你可以使用以下这些在线服务之一，它们提供了类似的用户体验：
```
[the remark this answers is not in the document]
Comment (202030481266) [reply README_CN.md:312]: OK, I will modify later.

File: README_CN.md [hunk 11]
```diff
@@ -252,106 +252,106 @@ sequence length and can be extended to 32K during inference time.
...
-- Yi APIs (Yi official)
-  - [Early access has been granted](https://x.com/01AI_Yi/status/1735728934560600536?s=20) to some applicants. Stay tuned for the next round of access!
+- Yi APIs (Yi 官方)
+  - [早期访问](https://x.com/01AI_Yi/status/1735728934560600536?s=20)的部分申请者已经获得了访问权限。敬请期待下一轮访问申请的信息！ 
 
-- [Yi APIs](https://replicate.com/01-ai/yi-34b-chat/api?tab=nodejs) (Replicate)
+- [Yi APIs](https://replicate.com/01-ai/yi-34b-chat/api?tab=nodejs) (Replicate，第三方网站)
 
-##### 🙋‍♀️ Run Yi in playground
+##### 🙋‍♀️ 在游乐场使用 Yi
```
[the remark this answers is not in the document]
Comment (202030481266) [reply README_CN.md:301]: playground一般是专业名词，和web demo一样，本来打算是不翻译的

**Comment by 202030481266** [comment]
I have already modified the translation by all the conversations. @GloriaLee01

**Pull request merged**